gradle/libs.versions.toml

```diff
@@ -52,7 +53,7 @@ toml4j = "0.7.2"
 
 # Plugin versions
 spotless = "6.9.1"
-spring-boot = "2.6.14"
+spring-boot = "2.7.5"
```

Comment:
I don't remember the exact reason. Something broke in service runner when using kotlin ref server, because of versions clash. Spring needs to be 2.7.x for latest coroutines
I can update to 2.7.9